tests/dataset/test_aoi.py

```diff
@@ -95,7 +95,6 @@ def test_stac_input(self):
         aoi = AOI(
             raster=row['tif'], label=row['gpkg'], split=row['split'],
             raster_bands_request=bands)
-        assert aoi.raster_bands_request == bands
```

Comment:
why remove it ?

Reply:
Good catch, I removed it for testing before pulling recent changes in 'develop'. I would revert back